client/src/components/RatingsAndReviews/AddReviewModal.jsx

```diff
@@ -100,6 +110,7 @@ class AddReviewModal extends React.Component {
           type="text"
           id="review-summary"
           name="reviewSummary"
+          placeholder={this.state.reviewSummary}
           defaultValue={this.state.reviewSummary}
```

Comment:
I think with the placeholder you can get rid of defaultValue. teh defaultValue will send and be validated as is even if the user puts in nothing